src/flows/components/PipeList.tsx

```diff
@@ -23,16 +20,7 @@ const PipeList: FC = () => {
     return <FlowPipe key={`pipe-${id}`} id={id} />
   })
 
-  return (
```

Comment:
if you're gonna kill this, kill the scroll context as well

Reply:
Burn it all!!